src/main/java/IP/duke/task/Event.java

```diff
@@ -0,0 +1,88 @@
+package IP.duke.task;
+
+import IP.duke.main.Date;
+
+import java.text.ParseException;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Represents tasks with specific timing.
+ * 
+ * @author Gordon Yit
+ * @version CS2103T, Semester 2
+ */
+public class Event extends Task {
+    private final String TASK_MARKER = "E";
```

Comment:
Same as what @eltonyeh mentioned, if your var is capitalised (you're declaring it as a constant) I think it should be declared as `static final`!